### ISSUE TITLE
This plugin is able to run unexpected JavaScript.

### ISSUE THREAD

**Issue by ghost** [zorgzerg/brackets-minimap#23]
Hi,

I found this issue by accident.
Contents of plain text(*.txt) are not escaped in Minimap.
### The following script is available to run.

```
<script>alert('Hello world!');</script>
```

![b6b1562ae37705f315fabd3cfbca1298](https://cloud.githubusercontent.com/assets/12760766/9690945/5286262e-537a-11e5-8395-ba732c396a86.png)
### Brackets will be crashed by the following script.

```
<script>location.href=''</script>
```

![ff37137735297de2e7e73557555f2fdd](https://cloud.githubusercontent.com/assets/12760766/9690964/654f51e0-537a-11e5-8f21-7d6bdefe8784.png)
### The following script would open other file.

![47955f57f095f7e0fba5fe531807612c](https://cloud.githubusercontent.com/assets/12760766/9691345/d5451780-537c-11e5-9e9c-c56630916f7a.png)

```
<script>location.href='file:///path/to/otherfile'</script>

// otherfile
<script>alert('Other File!');</script>
```

I'm being helped by Minimap very much.
So I hope Minimap will be more better !

Thank you !
(I hope you will forgive me that I'm not good at English...)

**Comment by zorgzerg** [comment]
OMG!!! It's realy is (( 
I try to understand and fix it
thx 4 issue

**Comment by zorgzerg** [comment]
This will be fixed in v3.2.6. Coming soon
